source/v0/TCPAsynch.cpp

```diff
@@ -51,8 +54,8 @@ socket_error_t TCPAsynch::open(const socket_address_family_t af, const socket_pr
 TCPAsynch::~TCPAsynch()
 {
     _TCPSockets--;
-    if (!_TCPSockets) {
+    if (_TCPSockets == 0 && _tick_handle != NULL) {
         minar::Scheduler::cancelCallback(_tick_handle);
-        _tick_handle = 0;
+        _tick_handle = NULL;
```

Comment:
In C++ these two statements are guaranteed to be equivalent.

Now that we're using C++11, this should probably be ```nullptr```.